corehq/form_processor/models.py

```diff
@@ -258,9 +259,7 @@ class AbstractAttachment(models.Model):
 
     @property
     def filepath(self):
-        if getattr(settings, 'IS_TRAVIS', False):
-            return os.path.join('/home/travis/', str(self.attachment_id))
-        return os.path.join('/tmp/', str(self.attachment_id))
+        return os.path.join(gettempdir(), str(self.attachment_id))
```

Comment:
@benrudolph what do you make of this from the travis build log (see also https://github.com/dimagi/commcare-hq/commit/66e32759129f2212f8c88fe0b029a7a8c54ce3f8)?
```
$ python .travis/testtmp.py
tempfile.tempdir -> None
tempfile.gettempdir() -> /tmp

exists(/tmp) -> True
write success: /tmp/ddd33f3792334cd3ac95c270c3e61fc7

exists(/home/travis/tmp) -> False
```